schema/resolvers.js

```diff
@@ -348,6 +327,57 @@ const Resolvers = {
       ctx.repositories.Answer.getById(answerId)
   },
 
+  ValidationType: {
+    __resolveType: answer => {
+      const validationEntity = getValidationEntity(answer.type);
+
+      switch (validationEntity) {
+        case "number":
+          return "NumberValidation";
```

Comment:
Going back to my first comment about lookup of validation type from the answer type we could eliminate the need to switch here altogether if we had something like:

```json
{
  "Number": "NumberValidation",
  "Currency": "NumberValidation"
}
```

Here we could just do this (and getValidationEntity would become redundant):

```js
return answerTypeMap[answer.type];
```

Reply:
Again probably leave it for now, but worth noting for future refactoring.